vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -296,6 +304,44 @@ testFips() {
   echo "$test:Finish"
 }
 
+testCloudInit() {
+  test="testCloudInit"
+  echo "$test:Start"
+  os_sku=$1
+
+  # Limit this test only to Mariner or Azurelinux 
+  if [[ "$os_sku" == "CBLMariner" || "$os_sku" == "AzureLinux" ]]; then
+    echo "Checking if cloud-init.log exists..."
+    FILE=/var/log/cloud-init.log
+    if test -f "$FILE"; then
+      echo "Cloud-init log exists. Checking its content..."
+      grep 'WARNING\|ERROR' $FILE | while read -r msg; do 
+        for pattern in "${CLOUD_INIT_LOG_IGNORING_MSG[@]}"; do
+            if [[ "$msg" == *"$pattern"* ]]; then
+                echo "Find WARNING/ERROR message: $msg in ignoring list, continue..."
```

Comment:
nit: Clumsy error message.
```suggestion
                echo "Find WARNING/ERROR message: $msg in ignoring list, continue..."
```
```suggestion
                echo "Ignoring WARNING/ERROR message from ignore list; '${msg}'."
```